net/tcp/tcp_recvwindow.c

```diff
@@ -328,7 +328,7 @@ bool tcp_should_send_recvwindow(FAR struct tcp_conn_s *conn)
    * E.g. consider the path MTU
    */
 
-  mss = tcp_rx_mss(dev);
+  mss = conn->mss;
```

Comment:
my understanding is:
* conn->mss is mss advertized by the peer via TCP_OPT_MSS.
* tcp_rx_mss is mss we advertize to the peer via TCP_OPT_MSS.

if these changes are intentional, i guess it needs a bit more explanations.

Reply:
@yamt 
I cancel the changes to this file.